features/bevragen/zoek-met-postcode-en-huisnummer/dev/gba.feature

```diff
@@ -0,0 +1,29 @@
+#language: nl
+
+@gba
+Functionaliteit: Zoek met postcode en huisnummer (GBA)
+
+Rule: Postcode is niet hoofdlettergevoelig
```

Comment:
Wijzig in:

`Rule: Postcode is niet hoofdlettergevoelig en mag zowel met als zonder spatie tussen het cijfer en letterdeel worden verstrekt.`